.ci/integration_test

```diff
@@ -158,6 +175,8 @@ EOF
 }
 
 function create_aws_secret() {
+  apt install -y pip > /dev/null
```

Comment:
Please add ` apt update`,  since this will be invoked by `run_test_as_processes`

Reply:
Addressed in [8e796da](https://github.com/gardener/etcd-backup-restore/pull/791/commits/8e796da2d8dd7f81265162e7bccc9ce8f7aa6279)